kolibri/plugins/facility_management/assets/src/views/UserPage/index.vue

```diff
@@ -48,11 +43,38 @@
       </template>
     </UserTable>
 
+    <nav>
+      <span dir="auto">
+        {{ $tr('pagination', { visibleStartRange, visibleEndRange, numFilteredUsers }) }}
+      </span>
+      <UiIconButton
+        type="primary"
+        :ariaLabel="$tr('previousResults')"
+        :disabled="pageNum === 1"
+        size="small"
+        class="pagination-button"
+        :dir="isRtl ? 'rtl' : 'ltr'"
+        @click="goToPage(pageNum - 1)"
+      >
+        <mat-svg v-if="isRtl" name="chevron_right" category="navigation" />
+        <mat-svg v-else name="chevron_left" category="navigation" />
+      </UiIconButton>
+      <UiIconButton
+        type="primary"
+        :ariaLabel="$tr('nextResults')"
+        :disabled="pageNum === 0 || pageNum === numPages"
+        size="small"
+        class="pagination-button"
+        :dir="isRtl ? 'rtl' : 'ltr'"
+        @click="goToPage(pageNum + 1)"
+      >
+        <mat-svg v-if="isRtl" name="chevron_left" category="navigation" />
+        <mat-svg v-else name="chevron_right" category="navigation" />
```

Comment:
instead of using `v-if`/`v-else`, recommend using the `is-rtl` global css class to flip the icon conditionally